style.css

```diff
@@ -770,4 +791,31 @@ textarea:hover {
   .multi {
     display: none;
   }
+
+.live-prg {
+  width: 424;
+  font-family: "Inter", sans-serif;
+font-style: normal;
+font-weight: normal;
+font-size: 16px;
+line-height: 30px;
+padding-top: 13px;
+height: 326px;
+}
+.container-pop {
+background-color: #fff;
+height: 631px;
+z-index: 1;
+width: 1156px;
```

Comment:
It is not always advicable to use hard coded widths. This may cause overflows on your design.
It is always better to set widths using percentages as this would even improve responsiveness.